components/layout_2020/flow/inline.rs

```diff
@@ -659,13 +659,20 @@ fn layout_atomic(
                 containing_block_for_children.style.writing_mode,
                 "Mixed writing modes are not supported yet"
             );
-            // FIXME: Do we need to adjust the static position of the hoisted fragments in the positioning
-            // context somewhere near here?
+
+            let collects_for_nearest_positioned_ancestor = ifc
+                .positioning_context
+                .collects_for_nearest_positioned_ancestor();
+            let mut positioning_context =
```

Comment:
Nit: `layout_block_level_children_in_parallel` and `layout_block_level_children_sequentially` use the name `child_positioning_context`, which may be slightly clearer than `positioning_context`

Reply:
Good point. I'll make this change.